### PR TITLE
Deduplicate extracted source cases

### PR DESCRIPTION
**Pull request by codebeige**

### CODE REVIEW

**Comment by codebeige** [comment]
@danse

**Comment by codebeige** [comment]
Updated change log

File: lib/group-by-id.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'use strict'
+
+
+function groupById (aggregator) {
```
Comment (danse) [inline lib/group-by-id.js]: why is this function empty?

Comment (codebeige) [inline lib/group-by-id.js]: Nice catch. :8ball:  This is a deprecated stub. I will remove it.

**Comment by danse** [comment]
i could not go very deep in understanding the logic, but it looks fine. and about the style, this is very fine Javascript :+1:

i especially appreciate the extensive test coverage :)

**Comment by codebeige** [comment]
:blush: Thank you.

**Pull request merged**